src/nbl/video/vulkan/profiles/platforms/android.json

```diff
@@ -1,4083 +1,4281 @@
-                "VK_KHR_shader_draw_parameters": 1,
-                "VK_KHR_shader_float16_int8": 1,
-                "VK_KHR_shader_float_controls": 4,
-                "VK_KHR_shader_integer_dot_product": 1,
-                "VK_KHR_shader_non_semantic_info": 1,
-                "VK_KHR_shader_subgroup_extended_types": 1,
-                "VK_KHR_shader_terminate_invocation": 1,
-                "VK_KHR_shared_presentable_image": 1,
-                "VK_KHR_spirv_1_4": 1,
-                "VK_KHR_storage_buffer_storage_class": 1,
-                "VK_KHR_swapchain": 70,
-                "VK_KHR_synchronization2": 1,
-                "VK_KHR_timeline_semaphore": 2,
-                "VK_KHR_uniform_buffer_standard_layout": 1,
-                "VK_KHR_variable_pointers": 1,
-                "VK_KHR_vulkan_memory_model": 3,
-                "VK_KHR_zero_initialize_workgroup_memory": 1
-            },
-            "features": {
-                "VkPhysicalDevice4444FormatsFeaturesEXT": {
-                    "formatA4R4G4B4": true,
-                    "formatA4B4G4R4": true
-                },
-                "VkPhysicalDeviceASTCDecodeFeaturesEXT": {
-                    "decodeModeSharedExponent": true
-                },
-                "VkPhysicalDeviceBorderColorSwizzleFeaturesEXT": {
-                    "borderColorSwizzle": true,
-                    "borderColorSwizzleFromImage": true
-                },
-                "VkPhysicalDeviceCustomBorderColorFeaturesEXT": {
-                    "customBorderColors": true,
-                    "customBorderColorWithoutFormat": true
-                },
-                "VkPhysicalDeviceDepthClampZeroOneFeaturesEXT": {
-                    "depthClampZeroOne": true
-                },
-                "VkPhysicalDeviceDeviceMemoryReportFeaturesEXT": {
-                    "deviceMemoryReport": true
-                },
-                "VkPhysicalDeviceExtendedDynamicState2FeaturesEXT": {
-                    "extendedDynamicState2": true,
-                    "extendedDynamicState2LogicOp": false,
-                    "extendedDynamicState2PatchControlPoints": true
-                },
-                "VkPhysicalDeviceExtendedDynamicStateFeaturesEXT": {
-                    "extendedDynamicState": true
-                },
-                "VkPhysicalDeviceFeatures": {
-                    "alphaToOne": false,
-                    "depthBiasClamp": true,
-                    "depthBounds": false,
-                    "depthClamp": true,
-                    "drawIndirectFirstInstance": true,
-                    "dualSrcBlend": false,
-                    "fillModeNonSolid": false,
-                    "fragmentStoresAndAtomics": true,
-                    "fullDrawIndexUint32": true,
-                    "geometryShader": true,
-                    "imageCubeArray": true,
-                    "independentBlend": true,
-                    "inheritedQueries": false,
```

Comment:
who's responsible for this?

Reply:
`geometryShader` is set to `false` in Apple and `inheritedQueries` is set to `false` in Pixel 6A

Reply:
it was also the `imageCubeArray` that was worrysome